<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookInfoView</class>
 <widget class="QWidget" name="BookInfoView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QWidget" name="bannerBackground" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#bannerBackground {
	background-image: url(:/branding/images/brand/background.jpg);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,0">
      <property name="rightMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="bannerLogo">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/branding/images/brand/logoSmall.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="bannerTitle">
        <property name="styleSheet">
         <string notr="true">#bannerTitle {
	color: rgb(255, 255, 255);
	font: 700 60pt &quot;Segoe UI&quot;;
}</string>
        </property>
        <property name="text">
         <string>City Library</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="logoutButton">
        <property name="text">
         <string>Log Out</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="bookInfoScroll">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1063</width>
        <height>534</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="backButton">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="text">
          <string>back</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="bookInfoLayout">
         <item>
          <widget class="QWidget" name="bookCoverLayout" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="coverLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="bookDescriptionLayout" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="titleLabel">
              <property name="text">
               <string>title</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
              <item>
               <layout class="QGridLayout" name="bookDetailsLayout">
                <item row="1" column="1">
                 <widget class="QLabel" name="isbnOutputLabel">
                  <property name="text">
                   <string>placeHolder</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="isbnLabel">
                  <property name="text">
                   <string>ISBN:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="authorOutputLabel">
                  <property name="text">
                   <string>placeHolder</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="authorLabel">
                  <property name="text">
                   <string>Author:</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QStackedWidget" name="bookOptionStackedWidget">
                  <property name="currentIndex">
                   <number>0</number>
                  </property>
                  <widget class="QWidget" name="adminPage">
                   <layout class="QVBoxLayout" name="verticalLayout_8">
                    <item>
                     <widget class="QPushButton" name="pushButton_2">
                      <property name="text">
                       <string>Edit Book</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton">
                      <property name="text">
                       <string>Delete Book</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                  <widget class="QWidget" name="memberPage"/>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QWidget" name="widget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QTextBrowser" name="bookDescription"/>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QStackedWidget" name="stackedWidget">
                   <widget class="QWidget" name="page"/>
                   <widget class="QWidget" name="page_2"/>
                  </widget>
                 </item>
                 <item>
                  <widget class="QStackedWidget" name="stackedWidget_2">
                   <widget class="QWidget" name="page_3"/>
                   <widget class="QWidget" name="page_4"/>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="footerBackground" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>110</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>110</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
          <item row="0" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="footerLogo">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="images.qrc">:/branding/images/brand/logoSmall.png</pixmap>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>472</width>
              <height>5</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
